app/server.py

```diff
@@ -95,30 +114,30 @@ async def get_appraisal(
     log_level = query_dict.get("log_level", "INFO")
     LOGGER.setLevel(LOGGER.setLevel(logging._nameToLevel[log_level]))
     message = query_dict["message"]
+    qid = str(uuid4())[:8]
     if "result" not in message:
         return JSONResponse(content={"status": "Rejected",
                                      "description": "No Results.",
                                      "job_id": qid}, status_code=200)
     callback = query_dict["callback"]
-    qid = str(uuid4())[:8]
     background_tasks.add_task(appraise, qid, message, callback)
     return JSONResponse(content={"status": "Accepted",
                                  "description": f"Appraising answers. Will send response to {callback}",
                                  "job_id": qid}, status_code=200)
 
 @APP.post("/sync_get_appraisal", response_model=Response)
 async def sync_get_appraisal(
-    query: Query
+    query: Query = Body(..., example=EXAMPLE)
 ):
     query_dict = query.dict()
-    log_level = query_dict.get("log_level", "INFO")
-    LOGGER.setLevel(LOGGER.setLevel(logging._nameToLevel[log_level]))
+    log_level = query_dict["log_level"] or "INFO"
+    LOGGER.setLevel(logging._nameToLevel[log_level])
     message = query_dict["message"]
+    qid = str(uuid4())[:8]
     if "result" not in message:
         return JSONResponse(content={"status": "Rejected",
                                      "description": "No Results.",
                                      "job_id": qid}, status_code=200)
```

Comment:
A 200 status code here doesn't make sense if it's rejected.

Reply:
Changing this will also break your test ;)

Reply:
IMO, this could just be a 400.

Reply:
This is an interesting case, I think. On one hand, this isn't strictly invalid, so I don't think throwing a 400 would be right. But I do think a flag should be thrown to show that there are no results, which was why I decided to do it this way. Maybe change the status to "Accepted" but keep the description the same instead?

Reply:
I'm not sure I agree. Sure, it's not invalid TRAPI, but at this point in the query process and for what our service does, we need results to exist, even if it's an empty list. If it doesn't exist, I think someone should be notified in a more drastic way than some text in a message.

Reply:
You've convinced me